<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackViewer</class>
 <widget class="QMainWindow" name="TrackViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1299</width>
    <height>1035</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TrackViewer</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="Projection">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>11</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X-Z Projection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Next_ev">
    <property name="geometry">
     <rect>
      <x>1127</x>
      <y>240</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray
}

QPushButton:pressed {
	background-color: qlineargradient( x1: 0, y1: 0, x2: 0, y2: 1,
										stop: 0 #dadbde, stop: 1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>Next Event</string>
    </property>
   </widget>
   <widget class="QLabel" name="Projection_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>513</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Y-Z Projection</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>1121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>-11</y>
      <width>20</width>
      <height>1061</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>-9</x>
      <y>-16</y>
      <width>20</width>
      <height>1063</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Prev_ev">
    <property name="geometry">
     <rect>
      <x>1127</x>
      <y>190</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray
}

QPushButton:pressed {
	background-color: qlineargradient( x1: 0, y1: 0, x2: 0, y2: 1,
										stop: 0 #dadbde, stop: 1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>Previous Event</string>
    </property>
   </widget>
   <widget class="QLabel" name="event_num">
    <property name="geometry">
     <rect>
      <x>1127</x>
      <y>30</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>None</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="event_frame">
    <property name="geometry">
     <rect>
      <x>1127</x>
      <y>30</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="ev_num">
    <property name="geometry">
     <rect>
      <x>1123</x>
      <y>4</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Enter event number:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="show_pts">
    <property name="geometry">
     <rect>
      <x>1127</x>
      <y>400</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray
}

QPushButton:pressed {
	background-color: qlineargradient( x1: 0, y1: 0, x2: 0, y2: 1,
										stop: 0 #dadbde, stop: 1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>Show points</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_reconst">
    <property name="geometry">
     <rect>
      <x>1127</x>
      <y>450</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray
}

QPushButton:pressed {
	background-color: qlineargradient( x1: 0, y1: 0, x2: 0, y2: 1,
										stop: 0 #dadbde, stop: 1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>Show reconstruction</string>
    </property>
   </widget>
   <widget class="QLabel" name="draw_control">
    <property name="geometry">
     <rect>
      <x>1127</x>
      <y>373</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Drawing control</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ev_control">
    <property name="geometry">
     <rect>
      <x>1127</x>
      <y>170</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Event control</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGraphicsView" name="XZ_proj">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>0</y>
      <width>1082</width>
      <height>475</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid white;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="YZ_proj">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>500</y>
      <width>1082</width>
      <height>475</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid white;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="graph_1" native="true">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>22</y>
      <width>1013</width>
      <height>473</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QCustomPlot" name="graph_2" native="true">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>522</y>
      <width>1013</width>
      <height>473</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="up_frame">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>0</y>
      <width>1116</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="low_frame">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>500</y>
      <width>1116</width>
      <height>535</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="decor_frame">
    <property name="geometry">
     <rect>
      <x>1119</x>
      <y>0</y>
      <width>180</width>
      <height>1035</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="XZ_trig">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>0</y>
      <width>1082</width>
      <height>475</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid white;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="YZ_trig">
    <property name="geometry">
     <rect>
      <x>-80</x>
      <y>510</y>
      <width>1082</width>
      <height>475</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid white;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
   </widget>
   <zorder>up_frame</zorder>
   <zorder>graph_1</zorder>
   <zorder>decor_frame</zorder>
   <zorder>low_frame</zorder>
   <zorder>graph_2</zorder>
   <zorder>YZ_proj</zorder>
   <zorder>XZ_proj</zorder>
   <zorder>Projection</zorder>
   <zorder>Next_ev</zorder>
   <zorder>Projection_2</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>line_3</zorder>
   <zorder>Prev_ev</zorder>
   <zorder>event_frame</zorder>
   <zorder>show_pts</zorder>
   <zorder>show_reconst</zorder>
   <zorder>draw_control</zorder>
   <zorder>ev_control</zorder>
   <zorder>event_num</zorder>
   <zorder>ev_num</zorder>
   <zorder>XZ_trig</zorder>
   <zorder>YZ_trig</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
